pkg/secrets/filedriver/filedriver.go

```diff
@@ -0,0 +1,174 @@
+package filedriver
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/containers/storage/pkg/lockfile"
+	"github.com/pkg/errors"
+)
+
+// secretsDataFile is the file where secrets data/payload will be stored
+var secretsDataFile = "secretsdata.json"
+
+// ErrNoSecretData indicates that there is not data associated with an id
+var ErrNoSecretData = errors.New("No secret data with ID")
```

Comment:
Errors are by convention lower case.  This way, they can be wrapped.  Logs on the contrary should be capitalized so they're easier to read (e..g, `logrus.Errorf("Start of the sentence")`).